lib/components/artist/__tests__/header-tests.js

```diff
@@ -0,0 +1,21 @@
+import 'react-native'
+
+import React from 'react'
+import renderer from 'react-test-renderer'
+
+import Header from '../header'
+
+jest.mock('../../spinner.js', () => 'ARSpinner')
+
+it('renders properly', () => {
+  const artist = {
+      name: 'Marcel Duchamp',
+      nationality: 'French',
+      birthday: '11/17/1992',
+      counts: {
+        follows: 22,
```

Comment:
What are your thoughts on testing the follow button? or do we have examples of followed and unfollowed artists thoughout the snap suite? (I guess so, come to think of it)